en/lessons/basics/functions.md

```diff
@@ -1,425 +1,83 @@
 ---
-version: 1.2.0
+version: 1.0.0
 title: Functions
 ---
 
-In Elixir and many functional languages, functions are first class citizens.
-We will learn about the types of functions in Elixir, what makes them different, and how to use them.
+Functions are at the heart of every Haskell program. Let us now see how to use them.
```

Comment:
I think "Let us now see how to use them" lacks coherence / doesn't introduce new information to the reader. Perhaps you could talk about the nature of the article (e.g. "You'll see some basic examples of functions below").